crm_project_task/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Emilio Pascual (`Moduon <https://www.moduon.team/>`__)
```

Comment:
Check my issue and all explanations plus testing, etc.. I think I'm also contributor 😄 . Functional and consultants are really needed in OpenSource

https://github.com/readme/featured/open-source-non-code-contributions

Reply:
Yes, you are right. Functionals are  essentials.